

const DefaultData = () => {
    try{

    }catch (error){
      console.log("Error", error.message)
    }
}